packages/teuchos/numerics/src/Teuchos_LAPACK.cpp

```diff
@@ -1331,7 +1331,7 @@ namespace Teuchos
     std::vector<std::complex<float> > w (n);
     std::complex<float>* w_rawPtr = (n == 0) ? NULL : &w[0];
     GEEV (JOBVL, JOBVR, n, A, lda, w_rawPtr, VL, ldvl, VR, ldvr, WORK, lwork, RWORK, info);
-    if (info == 0) {
+    if (*info == 0) {
```

Comment:
What there no unit test to catch this bug?  Is there no unit test to catch this now?

Reply:
I agree that the state of `Teuchos::LAPACK` unit testing is poor, but I'm not sure if we have to fix that now.  On the other hand, if there's an easy way to write a regression test for this one issue, let's do it.

Reply:
As long s there is a test that had to change to notice this, that is fine for now.